utils/index.js

```diff
@@ -123,3 +123,7 @@ export const getStyleLog = style => {
 export const analyticsSendCollect = e => {
   require("./analytics").sendCollect(e);
 };
+
+export const analyticsSendBotCollect = e => {
```

Comment:
Any specific reason you do not import at the top of the file with `import`?

Reply:
Not. Do i fix it.